compiler/src/dotty/tools/dotc/CompilationUnit.scala

```diff
@@ -105,7 +106,7 @@ class CompilationUnit protected (val source: SourceFile) {
 
 object CompilationUnit {
 
-  class SuspendException extends Exception
+  class SuspendException extends Exception with NoStackTrace
```

Comment:
Sounds like `ControlThrowable` or maybe `ControlException`.

Reply:
ControlException doesn't seem to exist, ControlThrowable might make sense but that kind of change would be more appropriate for a separate PR.

Reply:
`ControlException` is the Odersky proposal, but as you say, out of scope.

Reply:
It's definitely not a `ConrolThrowable`. We want handlers to catch the exception.